airmozilla/popcorn/templates/popcorn/editor.html

```diff
@@ -23,6 +23,7 @@
 {% endblock %}
 
 {% block full_width_content %}
+<slug data-url="{{ slug }}"></slug>
```

Comment:
That means, just being part of the `<div id="editor">` tag it means that those data blocks are related to the editor thing.